src/content/contributor-docs/es/access.mdx

```diff
@@ -1,3 +1,9 @@
+---
+title: |
+  Nuestro enfoque en el acceso
+description: |
+  What our commitment to access means for contributors and users of the library.
+---
 {/* Lo que significa nuestro compromiso con el acceso para colaboradoras y usuarias de la biblioteca. */}
```

Comment:
@limzykenneth Hi Ken, do you mean line 7?

Reply:
No, line 5